Libraries/vendor/emitter/EventEmitter.js

```diff
@@ -168,6 +168,36 @@ class EventEmitter {
       this._currentSubscription = null;
     }
   }
+
+  /**
+   * Removes the given listener for event of specific type.
+   *
+   * @param {string} eventType - Name of the event to emit
+   * @param {function} listener - Function to invoke when the specified event is
+   *   emitted
+   *
+   * @example
+   *   emitter.removeListener('someEvent', function(message) {
+   *     console.log(message);
+   *   }); // removes the listener if already registered
+   *
+   */
+  removeListener(eventType: String, listener) {
```

Comment:
When you rebase, you'll notice that the implementation of the `EventEmitter.emit` method (on which this `removeListener` method seems to be based) has changed quite a lot. Please update to reflect that style, as the use of Object.keys to enumerate subscriptions (which is an array, not an Object) is a bit strange.